src/main/java/com/spotinst/sdkjava/model/SpotinstSubscriptionService.java

```diff
@@ -125,4 +135,80 @@ public static Boolean deleteSubscription(String subscriptionId, String authToken
 
         return retVal;
     }
+
+    public static List<ApiSubscriptionResponse> getAllSubscriptionEvents(String authToken, String account) throws SpotinstHttpException {
+
+        // Init retVal
+        List<ApiSubscriptionResponse> retVal = null;
+
+        // Get endpoint
+        SpotinstHttpConfig config = SpotinstHttpContext.getInstance().getConfiguration();
+        String apiEndpoint = config.getEndpoint();
+
+        // Get the headers for AWS.
+        Map<String, String> headers = buildHeaders(authToken);
+
+        // Build query params
+        Map<String, String> queryParams = new HashMap<>();
+
+        // Add account Id Query param
+        if (account != null) {
+            queryParams.put("accountId",account);
+        }
+
+        //Build URI
+        String uri = String.format("%s/events/subscription", apiEndpoint);
+
+        // Send the request.
+        RestResponse response = RestClient
+                .sendGet(uri, headers, queryParams);
+
+        // Handle the response.
+        SubscriptionApiResponse subscriptionApiResponse = getCastedResponse(response, SubscriptionApiResponse.class);
+        if (subscriptionApiResponse.getResponse().getStatus().getCode() == HttpStatus.SC_OK) {
+            retVal = subscriptionApiResponse.getResponse().getItems();
+        }
+
+        return retVal;
+    }
+
+    static Boolean updatSubscription(ApiSubscription subscriptionToUpdate, String subscriptionId, String authToken, String account) throws SpotinstHttpException {
```

Comment:
updateSubscription - 'e' missing in function name

Reply:
fixed